blocks/v2-article-search/v2-article-search.css

```diff
@@ -0,0 +1,221 @@
+:root {
+  --icon-size: 24px;
+  --bullets-gap: 16px 8px;
+}
+
+.v2-article-search-wrapper {
+  background-color: var(--background-color);
+}
+
+.v2-article-search.block {
+  width: var(--wrapper-width);
+  margin: 0 auto;
+}
+
+.v2-article-search__fieldset {
+  padding: 10px 20px;
+  border: none;
+  display: flex;
+  justify-content: space-between;
+  color: var(--c-grey-4);
+  height: 54px; /* Check if this is needed when padding/margin are adjusted */
+}
+
+.v2-article-search__filter-container {
+  position: relative;
+}
+
+.v2-article-search__filter-input,
+.v2-article-search__filter-dropdown {
+  display: flex;
+  gap: .3rem;
+  align-items: center;
+}
+
+.v2-article-search__filter-dropdown{
+  font-size: var(--f-button-font-size);
+  cursor: pointer;
+}
+
+.v2-article-search__filter-dropdown-icon {
+  font-size: var(--icon-size);
+}
+
+.v2-article-search__filter-dropdown--open {
+  visibility: hidden;
+}
+
+.v2-article-search__filter-dropdown--open .v2-article-search__filter-dropdown-icon {
+  transform: rotate(180deg);
+}
+
+.v2-article-search__filter-list-wrapper {
+  height: 100vh;
+  display: none;
+}
+
+ul.v2-article-search__filter-list {
```

Comment:
Missing the box-shadow here
`box-shadow: 0px 4px 8px 0px #00000033;`

Reply:
added the shadow and a little aesthetic fix for moblie